dialects/velox_presto.yaml

```diff
@@ -799,6 +799,12 @@ scalar_functions:
   - name: string_split
     local_name: "split"
     unsupported: True
+  - name: starts_with
+    unsupported: True
+  - name: ends_with
+    unsupported: True
+  - name: contains
+    unsupported: True
```

Comment:
Velox also supports these 3
https://facebookincubator.github.io/velox/functions.html

Could you try enabling it?

Reply:
I tried enabling it, but hit failure as unsupported function ([here](https://github.com/voltrondata/bft/actions/runs/9890689291/job/27319466588?pr=78)).
May be python binding is missing for these ?